packages/vm/src/vm.ts

```diff
@@ -20,6 +20,7 @@ import type {
 } from './types.js'
 import type { BlockchainInterface } from '@ethereumjs/blockchain'
 import type { EVMStateManagerInterface } from '@ethereumjs/common'
+import type { EVMInterface } from '@ethereumjs/evm/dist/cjs/types.js'
```

Comment:
These imports are still a total no-go for merging, I guess that's obvious?

Reply:
Huh, how did this get introduced? :thinking: I think VSCode auto-imported this

Reply:
Nvm see my new comment on the PR, PR was not yet ready to be reviewed (sorry for marking it as ready-to-review)